docs/weave-tutorial/03-codec.md

```diff
@@ -189,11 +199,11 @@ plugins:
 After defining your state and messages run `make protoc`. This script will download `prototool` docker image and compile codec.proto file `codec.pb.go` file.
 Now we have the scaffold of our application thanks to auto-generated _codec.pb.go_ file.
```

Comment:
`Is it really auto-generated if we compiled it ourselves in the previous step?`

Reply:
`Auto-generated` is actually equals to `compilation`. Auto-generation in this concept means compiler *auto-generates* the files without any outside help.